rclpy/src/rclpy/_rclpy.c

```diff
@@ -5093,10 +5094,19 @@ PyMODINIT_FUNC PyInit__rclpy(void)
     return NULL;
   }
 
+  RCLError = PyErr_NewExceptionWithDoc(
+    "_rclpy.RCLError",
+    "Thrown when there is an error in rcl.",
+    PyExc_RuntimeError, NULL);
```

Comment:
I think you should check against `NULL` before calling `PyModule_AddObject`.
```suggestion
    PyExc_RuntimeError, NULL);
    if (NULL == RCLError) {
      Py_DECREF(m);
      return NULL;
    }
```